woostarter/parts/header.html

```diff
@@ -1,23 +1,31 @@
-<div style="height:var(--wp--preset--spacing--30)" aria-hidden="true" class="wp-block-spacer"></div>
-<!-- /wp:spacer -->
+<div class="wp-block-group">
+    <!-- wp:spacer {"height":"var:preset|spacing|30"} -->
+    <div style="height:var(--wp--preset--spacing--30)" aria-hidden="true" class="wp-block-spacer"></div>
+    <!-- /wp:spacer -->
 
-<!-- wp:group {"align":"wide","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
-<div class="wp-block-group alignwide"><!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|50"}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
-<div class="wp-block-group"><!-- wp:site-title {"textAlign":"center"} /-->
+    <!-- wp:group {"align":"wide","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+    <div class="wp-block-group alignwide">
+        <!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|50"}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+        <div class="wp-block-group">
+            <!-- wp:site-title {"textAlign":"center"} /-->
 
-<!-- wp:navigation {"className":"order-1 md:order-0","style":{"spacing":{"blockGap":"var:preset|spacing|40"}},"layout":{"type":"flex","flexWrap":"wrap"}} /--><!-- wp:woocommerce/customer-account {"displayStyle":"text_only","iconStyle":"line","iconClass":"wc-block-customer-account__account-icon"} /--></div>
-<!-- /wp:group -->
+            <!-- wp:navigation {"className":"order-1 md:order-0","style":{"spacing":{"blockGap":"var:preset|spacing|40"}},"layout":{"type":"flex","flexWrap":"wrap"}} /-->
+        </div>
+        <!-- /wp:group -->
 
-<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"right"}} -->
-<div class="wp-block-group">
-<!-- wp:search {"label":"Search","showLabel":false,"buttonText":"Search","buttonPosition":"button-only","buttonUseIcon":true,"isSearchFieldHidden":true,"style":{"elements":{"link":{"color":{"text":"var:preset|color|theme-2"}}},"color":{"background":"#ffffff00"},"border":{"width":"0px","style":"none"},"layout":{"selfStretch":"fixed","flexSize":"40px"}},"textColor":"theme-2"} /-->
-<!-- wp:woocommerce/mini-cart /--></div>
-<!-- /wp:group --></div>
-<!-- /wp:group -->
+        <!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"right"}} -->
+        <div class="wp-block-group">
+            <!-- wp:search {"label":"Search","showLabel":false,"buttonText":"Search","buttonPosition":"button-only","buttonUseIcon":true,"isSearchFieldHidden":true,"style":{"elements":{"link":{"color":{"text":"var:preset|color|theme-2"}}},"color":{"background":"#ffffff00"},"border":{"width":"0px","style":"none"},"layout":{"selfStretch":"fixed","flexSize":"40px"}},"textColor":"theme-2"} /-->
+            <!-- wp:woocommerce/customer-account {"displayStyle":"icon_only","iconStyle":"line","iconClass":"wc-block-customer-account__account-icon"} /-->
```

Comment:
Is this line the only real change on this PR? it's hard to read with the reformatting

Reply:
@MaggieCabrera  That’s just a single-line change. Apart from that, all other modifications are purely formatting-related - nothing functional.

Reply:
Thanks for your contribution! The thing is, I think what is missing from the original issue is adding `"metadata":{"ignoredHookedBlocks":["woocommerce/customer-account"]}` to the navigation block to avoid woocommerce from hooking into it and adding the my account text